go.md file has hidden whitespace character that causes validation fail on some systems

The file here:

https://raw.githubusercontent.com/OBOFoundry/OBOFoundry.github.io/1e0151ee9585e78e03964dcd861e40c830b2fad1/ontology/go.md

Has a hidden whitespace (`0a`) character after the `CC BY 4.0`. This is not really visible looking at this URL in chrome (shows up as a space for me)

```
$ od -t a ontology/go.md
0000300    n   s   e   s   /   b   y   /   4   .   0   /  nl  sp  sp   l
0000320    a   b   e   l   :  sp   C   C  sp   B   Y  sp   4   .   0   <C2>
0000340    <A0>  nl   l   a   b   e   l   :  sp   G   O  nl   d   e   s   c
```

This is how it looks in emacs:

![image](https://user-images.githubusercontent.com/50745/58195838-bfac3100-7c7d-11e9-83aa-3e561996a1ea.png)

What is odd that travis passes and `make test` passes for @rctauber however, on my system (OS X) I get:

```
./util/validate-metadata.py tmp/unsorted-ontologies.yml reports/metadata-violations.tsv reports/metadata-grid.csv
Traceback (most recent call last):
  File "./util/validate-metadata.py", line 107, in validate_metadata
    jsonschema.validate(item, s)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/jsonschema/validators.py", line 541, in validate
    cls(schema, *args, **kwargs).validate(instance)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/jsonschema/validators.py", line 130, in validate
    raise error
jsonschema.exceptions.ValidationError: {'label': 'CC BY 4.0\xa0', 'logo': 'http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png', 'url': 'https://creativecommons.org/licenses/by/4.0/'} is not valid under any of the given schemas

Failed validating 'oneOf' in schema['properties']['license']:
    {'oneOf': [{'properties': {'label': {'enum': ['CC-BY',
                                                  'CC BY 4.0',
                                                  'CC-BY 4.0']},
                               'url': {'enum': ['http://creativecommons.org/licenses/by/4.0/',
                                                'https://creativecommons.org/licenses/by/4.0/']}}},
               {'properties': {'label': {'enum': ['CC-0',
                                                  'CC0',
                                                  'CC0 1.0 Universal',
                                                  'CC0 1.0']},
                               'url': {'enum': ['http://creativecommons.org/publicdomain/zero/1.0/',
                                                'https://creativecommons.org/publicdomain/zero/1.0/']}}},
               {'properties': {'label': {'enum': ['CC-BY',
                                                  'CC-BY 3.0',
                                                  'CC BY 3.0']},
                               'url': {'enum': ['http://creativecommons.org/licenses/by/3.0/',
                                                'https://creativecommons.org/licenses/by/3.0/']}}},
               {'properties': {'label': {'enum': ['CC BY-ND 4.0']},
                               'url': {'enum': ['http://creativecommons.org/licenses/by-nd/4.0/',
                                                'https://creativecommons.org/licenses/by-nd/4.0/']}}},
               {'properties': {'label': {'enum': ['CC-BY-SA',
                                                  'CC BY-SA 2.0']},
                               'url': {'enum': ['http://creativecommons.org/licenses/by-sa/2.0/',
                                                'https://creativecommons.org/licenses/by-sa/2.0/']}}},
               {'properties': {'label': {'enum': ['CC-BY',
                                                  'CC BY 2.0',
                                                  'CC-BY 2.0']},
                               'url': {'enum': ['http://creativecommons.org/licenses/by/2.0/',
                                                'https://creativecommons.org/licenses/by/2.0/']}}}],
     'properties': {'label': {'type': 'string'},
                    'url': {'format': 'uri', 'type': 'string'}},
     'required': ['url', 'label'],
     'type': 'object'}

On instance['license']:
    {'label': 'CC BY 4.0\xa0',
     'logo': 'http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png',
     'url': 'https://creativecommons.org/licenses/by/4.0/'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./util/validate-metadata.py", line 391, in <module>
    validate(sys.argv)
  File "./util/validate-metadata.py", line 27, in validate
    add = validate_metadata(item, schemas)
  File "./util/validate-metadata.py", line 149, in validate_metadata
    format_license_msg(search.group(1))
  File "./util/validate-metadata.py", line 184, in format_license_msg
    d = json.loads(substr.replace('\'', '"'))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Invalid \escape: line 1 column 21 (char 20)
make: *** [reports/metadata-grid.csv] Error 1
```

--- a/ontology/go.md
+++ b/ontology/go.md
@@ -8,7 +8,7 @@ contact:
   label: Suzi Aleksander
 license:
   url: https://creativecommons.org/licenses/by/4.0/
-  label: CC BY 4.0 
+  label: CC BY 4.0
 label: GO
 description: An ontology for describing the function of genes and gene products
 title: Gene Ontology

